scripts/test/set-e2e-env-vars.sh

```diff
@@ -3,27 +3,26 @@ set -euo pipefail
 
 # Check for required argument
 if [[ "$#" -ne 1 ]]; then
-    echo "Usage: $0 <WORKSPACE>"
-    exit 1
+  echo "Usage: $0 <WORKSPACE>"
+  exit 1
 fi
 
 WORKSPACE="$1"
 
 # Set domain
 if [[ "$WORKSPACE" = "ndr-test" || "$WORKSPACE" = "pre-prod" ]]; then
-    DOMAIN="national-document-repository.nhs.uk"
+  DOMAIN="national-document-repository.nhs.uk"
 else
-    DOMAIN="access-request-fulfilment.patient-deductions.nhs.uk"
+  DOMAIN="access-request-fulfilment.patient-deductions.nhs.uk"
 fi
 
 # Set environment variables
-export PDM_METADATA_TABLE="${WORKSPACE}_PDMDocumentMetadata"
+export PDM_METADATA_TABLE="${WORKSPACE}_COREDocumentMetadata"
```

Comment:
Change PDM to CORE also look into setting this in the code. rather than a env var. Make another ticket